include/abb_librws/v1_0/rws_progress.h

```diff
@@ -0,0 +1,25 @@
+#include <abb_librws/rws.h>
+#include <abb_librws/v1_0/rws_client.h>
+
+namespace abb :: rws :: v1_0 :: progress
+{
+    /**
+     * \brief A structure for containing progress resource information.
+     */
+    struct ProgressInfo
+    {
+        std::string state;
+        int code;
```

Comment:
Please describe what `state` and `code` are.

Reply:
```suggestion
        std::string state;  /** \brief Progress state of resource*/
        int code;           /** \brief Returned retcode*/
```